### ISSUE TITLE
Installation problems in SUSE

### ISSUE THREAD

**Issue by BraulioV** [wazuh/wazuh-packages#84]
Hi all,

there's a problem when you try to install the wazuh-agent v3.7.0 in SUSE 12 and SUSE 11. The installation fails because the file `/var/ossec/tmp/src/init/ossec-hids-suse.init` is missing.

Log sample:

```shellsession
sles12-sp1:/home/vagrant # zypper install wazuh-agent
Building repository 'SUSE-12.1 - Wazuh' cache .......................................................................................................................................................................................[done]Loading repository data...
Reading installed packages...
Resolving package dependencies...

The following NEW package is going to be installed:
  wazuh-agent

The following package is not supported by its vendor:
  wazuh-agent

1 new package to install.
Overall download size: 6.8 MiB. Already cached: 0 B. After the operation, additional 73.8 MiB will be used.
Continue? [y/n/? shows all options] (y): y
Retrieving package wazuh-agent-3.7.0-1.x86_64                                                                                                                                                         (1/1),   6.8 MiB ( 73.8 MiB unpacked)Retrieving: wazuh-agent-3.7.0-1.x86_64.rpm ............................................................................................................................................................................[done (514.8 KiB/s)]Checking for file conflicts: ........................................................................................................................................................................................................[done](1/1) Installing: wazuh-agent-3.7.0-1.x86_64 ........................................................................................................................................................................................[done]Additional rpm output:
install: cannot stat '/var/ossec/tmp/src/init/ossec-hids-suse.init': No such file or directory
wazuh-agent: unknown service
wazuh-agent: unknown service


sles12-sp1:/home/vagrant #
```

**Comment by BraulioV** [comment]
Working in this branch: [wazuh-packages/3.7-fix-suse](https://github.com/wazuh/wazuh-packages/tree/3.7-fix-suse).

**Comment by alberpilot** [comment]
Solved: #85 .
